doc/interop-test-descriptions.md

```diff
@@ -1391,22 +1404,38 @@ fully communicate metadata.
 ### Backend metrics report
 [Backend Metrics Report]: #backend-metrics-report
 
-Server reports backend metrics data in both per-query and out-of-band cases, with metrics data
-indicated from the unary call request.
+#### Report per-query metrics
+
+Server reports backend metrics data in per-query case, echoing metrics data from the unary call request.
 
 Using ORCA APIs we install the per-query metrics reporting server interceptor, so that it can attach 
 metrics per RPC.
-Also using ORCA APIs we register the `OpenRCAService` implementation to the server, so that it can
-report metrics periodically. The minimum report interval in the ORCA service is set to 1 sec.
 
-During test, the server will receive unary requests from the client that each may contain up to two
-test load report, indicating whether it needs to update metrics for the current call or at the OOB server.
+During `orca_per_query` test, the server will receive unary requests from the client that may contain a 
```

Comment:
Don't specify which test. The tests get to choose which features they use. Just say "If `SimpleRequest.orca_per_rpc_report` is set, the server will add the metric data from `orca_per_rpc_report` to the RPC using the language's CallMetricRecorder." You can still say something about "Both utilization and request cost metrics should be copied."